peps/pep-0602.rst

```diff
@@ -10,6 +10,7 @@ Type: Informational
 Content-Type: text/x-rst
 Created: 04-Jun-2019
 Python-Version: 3.9
+Post-History: `09-Oct-2023 <https://discuss.python.org/t/27002>`__
```

Comment:
Shouldn't there be just one trailing underscore in reST?

```suggestion
Post-History: `09-Oct-2023 <https://discuss.python.org/t/27002>`_
```

Reply:
Double underscore is an "anonymous link":

> Note the two underscores at the end of the reference. This creates an anonymous link, which means the same URL cannot later be referenced using only its anchor text. 

https://developer.lsst.io/restructuredtext/style.html

```console
❯ rg "Post-History.*\`__,$" | wc -l
      42

❯ rg "Post-History.*\`_,$" | wc -l
       0
```